src/plugins/hash-len-suggest.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-console */
+
+/*
+at webpack settings:
+const cssHashLen = 8
+...
+{
+    loader: 'css-loader',
+    options: {
+        modules: {
+            localIdentName: `[hash:base64:${cssHashLen}]`,
+            getLocalIdent: MyOneLetterCss.getLocalIdent
+        }
+    }
+}
+...
+plugins: [
+        ...plugins,
+        new HashLenSuggest({
+            instance: MyOneLetterCss,
+            selectedHashLen: cssHashLen
+        })
+    ]
+*/
+
+class HashLenSuggest {
+  constructor({ instance, selectedHashLen }) {
+    this.instance = instance;
+    this.selectedHashLen = selectedHashLen;
+  }
+
+  apply(compiler) {
+    compiler.plugin('done', this.run);
+  }
+
+  static collectHashLen(data) {
+    const matchLen = {};
+    const base = {};
+
+    Object.values(data).forEach(({ name }) => {
+      for (let len = 1; len <= name.length; len += 1) {
+        base[len] = base[len] || {};
+        const hash = name.substr(0, len);
+
+        if (base[len][hash]) {
+          matchLen[len] = matchLen[len] || 0;
+          matchLen[len] += 1;
+        } else {
+          base[len][hash] = 1;
+        }
+      }
+    });
+
+    return matchLen;
+  }
```

Comment:
We should use weak cache here, otherwise is was memory leak in wathc mode

Reply:
need help with it

Reply:
@alexander-akait help plz